beginner_source/scaled_dot_product_attention_tutorial.py

```diff
@@ -0,0 +1,320 @@
+"""
+An overview of torch.nn.functional.scaled_dot_product_attention
```

Comment:
Suggest motivating the title a little more wrt what you can do (e.g. build performant Transformer variations) - ideally you want to hook people in with the buzzwords :p

Reply:
replace the latest iteration (... implement variations....) with 
“Implement High-Performance Transformers with SCALED DOT PRODUCT ATTENTION”
or
“Implement High-Performance MultiHead-Attention with SCALED DOT PRODUCT ATTENTION”